Library/Homebrew/utils/shell_env.sh

```diff
@@ -0,0 +1,71 @@
+keepkeys=(
+HOMEBREW_*
+GEM_*
+HOME
+PWD
+LANG
+PATH
+)
+
+prune-shell-env() {
+  # Associative arrays are only a Bash 4 feature
+  #declare -A aEnv
+
+  local verbose
+  
+  local envPATH=`which env`
+  local echoPATH=`which echo`
```

Comment:
We'll set want to set the PATH to the default system value e.g. `/usr/bin:/bin:/usr/sbin:/sbin`